wallet-new/src/Cardano/Wallet/WalletLayer/Kernel.hs

```diff
@@ -125,6 +125,7 @@ bracketActiveWallet pm walletPassiveLayer passiveWallet walletDiffusion runActiv
           walletPassiveLayer = walletPassiveLayer
         , pay                = Active.pay          w
         , estimateFees       = Active.estimateFees w
+        , createUnsignedTx   = Active.createUnsignedTx w
```

Comment:
Let's align `w` vertically here :trollface:

Reply:
Maaaaaan. I'd better have none of them aligned to be honest, in the elm-style :trollface: